docs/README.md

```diff
@@ -1,49 +1,55 @@
 # Ratify Documentation
 
-This is an overview of the documentation available today for Ratify in the folders
+Within this directory you will find documentation for Ratify.  The documentation is broken down into the following sections:
 
-Titles by sub-folder in this directory
+## Quickstarts 
 
-## examples
+## End-to-end scenarios
 
-These walk through end-to-end examples of using ratify.
+- [ratify-on-aws](quickstarts/ratify-on-aws.md) - How to use ratify on AWS
+- [ratify-verify-azure-cmd](quickstarts/ratify-on-azure.md) - Use notation, oras, and ratify to build and sign a container image and SBOM and verify it with the ratify cli.
+- [working-with-spdx](quickstarts/working-with-spdx.md) - Use ratify with syft, oras, and verification using ratify cli.
 
-- [ratify-on-aws](examples/ratify-on-aws.md) - How to use ratify on AWS
-- [ratify-verify-azure-cmd](examples/ratify-verify-azure-cmd.md) - Use notation, oras, and ratify to build and sign a container image and SBOM and verify it with the ratify cli.
-- [working-with-spdx](examples/working-with-spdx.md) - Use ratify with syft, oras, and verification using ratify cli.
+### Plugin development:
 
-## reference
+- [creating-plugins](quickstarts/creating-plugins.md) - Details on creating your own plugins for use with ratify.
 
-These documents can be found generally useful to understand using or implementing ratify, but do not walk through end-to-end examples.
+### Policy Management:
 
-- [usage](reference/usage.md) - Additional information for using the `ratify` executable
-- [gatekeeper-policy-authoring](reference/gatekeeper-policy-authoring.md) - Authoring gatekeeper policies for use with ratify, including rego references/examples.
-- [oras-auth-provider](reference/oras-auth-provider.md) - Explanation of various authentication mechanisms available for use with ratify.
-- [creating-plugins](reference/creating-plugins.md) - Details on creating your own plugins for use with ratify.
-- [CRD references](reference/crds/) - Describes the required and optional properties of ratify CRDs
-- [instrumentation](reference/instrumentation.md) - Details on the current supported instruments and metric provider setup guides
-## developer
+- [gatekeeper-policy-authoring](quickstarts/gatekeeper-policy-authoring.md) - Authoring gatekeeper policies for use with ratify, including rego references/examples.
+
+### SPDX Integration:
+
+- [working-with-spdx](quickstarts/working-with-spdx.md) - Use ratify with syft, oras, and verification using ratify cli.
```

Comment:
The title "working with spdx" , and description is the same as the link in "working with spdx" with in the end-to-end scenario, should we update the description to indicate how they are different?

Reply:
That's an oversight on my end I'll remove the ### SPDX integration section and leave the link in the End-to-end. :)